target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vgv952cjw33-e-ir.dtsi

```diff
@@ -0,0 +1,419 @@
+		};
+		out_1 {/*Unknown*/
+			gpio-export,name = "hc595_1";
+			gpio-export,output = <1>;
+			gpios = <&hc595 1 GPIO_ACTIVE_HIGH>;
+		};
+
+		// out_2 is used to reset touch IC and owned by eb904_keypad
+
+		out_3 {/*Unknown*/
+			gpio-export,name = "hc595_3";
+			gpio-export,output = <1>;
+			gpios = <&hc595 3 GPIO_ACTIVE_HIGH>;
+		};
+		out_dsl_eth { /* Switches DSL line + 100MiB eth or 1GiB ethernet on DSL/WAN input. Relay 1.*/
+			gpio-export,name = "dsl_en";
+			gpio-export,output = <1>;
+			gpios = <&hc595 4 GPIO_ACTIVE_LOW>;
+		};
+		out_5 { /* Switches filter for DSL line On/Off. Relays 2 and 3 simultaneously.*/
+			gpio-export,name = "annexj";
+			gpio-export,output = <1>;
+			gpios = <&hc595 5 GPIO_ACTIVE_HIGH>;
+		};
+		out_6 { /* Switch DSL line betwean ISDN modem and Si3050. Relay 5.*/
+			gpio-export,name = "annexb";
+			gpio-export,output = <1>;
+			gpios = <&hc595 6 GPIO_ACTIVE_HIGH>;
+		};
+		out_7 { /* Switch phone outputs N and F betwean analog DSL line and XS1 output from Lantiq SLIC. Relay 4.*/
+			gpio-export,name = "annexa";
+			gpio-export,output = <1>;
+			gpios = <&hc595 7 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	i2c {
+		compatible = "i2c-gpio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		gpios = <&gpio 19 GPIO_ACTIVE_HIGH /* sda */
+			&gpio 14 GPIO_ACTIVE_HIGH /* scl */
+		>;
+		//i2c-gpio,sda-open-drain;
+		//i2c-gpio,scl-open-drain;
+		i2c-gpio,delay-us = <5>;
+		/* Add touch panel Support */
+		tp: eb904tp@0x14 {
+			compatible = "lantiq,eb904_keypad";
+			reg = <0x14>;
+			interrupt-parent = <&icu0>;
+			interrupts = <135>;
+			eb904,interrupt-gpio = <&gpio  0 GPIO_ACTIVE_HIGH /* EXIN */>;
+			eb904,ctrl-rst-gpio = <&hc595 2 GPIO_ACTIVE_LOW /* rst */>;
+			eb904,alphas = /bits/ 8
+				 <0x07 /* left */
+				  0x0a /* down */
+				  0x0a /* right */
+				  0x0a /* ok */
+				  0x07 /* up */
+			>;
+			keypad,num-rows = <3>;
+			keypad,num-columns = <3>;
+			linux,keymap = <
+				    MATRIX_KEY(0x0, 0x1, KEY_UP)         /* ROW0, COL1 */
+				    MATRIX_KEY(0x1, 0x0, KEY_LEFT)       /* ROW1, COL0 */
+				    MATRIX_KEY(0x1, 0x1, KEY_ENTER)      /* ROW1, COL1 */
+				    MATRIX_KEY(0x1, 0x2, KEY_RIGHT)      /* ROW1, COL2 */
+				    MATRIX_KEY(0x2, 0x1, KEY_DOWN)       /* ROW2, COL1 */
+				    >;
+		};
+	};
+
+	mdio: mdio {
+		compatible = "lantiq,xrx200-mdio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		phy13: ethernet-phy@13 {
+			reg = <0x13>;
+			compatible = "lantiq,phy11g", "ethernet-phy-ieee802.3-c22";
+		};
+	};
+
+	rtl8367b {
+		compatible = "realtek,rtl8367b";
+		//gpio-sda = <&gpio0 1 0>;
+		//gpio-sck = <&gpio0 2 0>;
+		//cpu_port = <7>;
+		//realtek,extif0 = <1 0 1 1 1 1 1 1 2>; // default found on other profiles
+		realtek,extif1   = <1 0 1 1 0 0 1 1 2>; // based on vendor uboot-config
+		mii-bus = <&mdio>;
+	};
+};
+
+&vmmc {
+	status = "okay";
+	gpios = <&gpio 37 GPIO_ACTIVE_HIGH>; //reset_slic!
+};
+
+&localbus {
+    //	#address-cells = <2>;					// Just for info. From vr9.dtsi
```

Comment:
Please remove your debugging comments.

Reply:
Not all i have reduce them and let all usefull comments